ci/gdk_build.template.steps.yaml

```diff
@@ -18,7 +18,7 @@ common: &common
   retry:
     automatic:
       - <<: *agent_transients
-  timeout_in_minutes: 60
+  timeout_in_minutes: 120
```

Comment:
Where does this variable come from?

Reply:
I increased the timeout for the buildkite job, because a full test run with everything enabled can take up to 55 minutes, which is dangerously close to the 60 minutes here.